ibexa/commerce/5.0/config/packages/security.yaml

```diff
@@ -143,6 +143,13 @@ security:
             custom_authenticators:
                 - Ibexa\Rest\Security\Authenticator\RestAuthenticator
 
+        ibexa_fragment:
+            pattern: ^/_fragment
+            user_checker: Ibexa\Core\MVC\Symfony\Security\UserChecker
+            provider: ibexa
+            custom_authenticators:
+                - Ibexa\PageBuilder\Security\EditorialMode\FragmentAuthenticator
```

Comment:
Cannot we simply, I dunno...

Add this as a custom authenticator to `ibexa_front`?

Reply:
This seems to be working even better. It fixes issue with dashboards as the fallback `form_login` kicks in which is expected. Thanks!